app/models/inventory.rb

```diff
@@ -0,0 +1,3 @@
+class Inventory < ApplicationRecord
+  belongs_to :User
```

Comment:
The Inventory model should have a "has_many" association with inventory_food model based on the ER Diagram.